src/popup.js

```diff
@@ -16,7 +16,7 @@ function click(e) {
 		console.log("Add new parameter");
 		add_new_parameter();
 	} else if(e.target.id=="export"){
-		export_parameters_list("csv");
+		export_parameters_list("json");
```

Comment:
Hey @jatinAroraGit can you revert this to csv please? it will be fixed on the next PR

Reply:
thanks